vamb/__main__.py

```diff
@@ -1195,48 +1208,86 @@ def cluster_and_write_files(
         cuda=cuda,
         rng_seed=seed,
     )
-    # This also works correctly when max_clusters is None
+
     clusters = itertools.islice(cluster_generator, cluster_options.max_clusters)
-    cluster_dict: dict[str, set[str]] = dict()
 
     # Write the cluster metadata to file
     with open(Path(base_clusters_name + "_metadata.tsv"), "w") as file:
-        print("name\tradius\tpeak valley ratio\tkind\tbp\tncontigs\tmedoid", file=file)
-        for i, cluster in enumerate(clusters):
-            cluster_dict[str(i + 1)] = {
-                sequence_names[cast(int, i)] for i in cluster.members
-            }
+        unsplit_path = Path(base_clusters_name + "_unsplit.tsv")
+        split_path = Path(base_clusters_name + "_split.tsv")
+        with (
+            open(unsplit_path, "a") as unsplit_clusters_file,
+            open(split_path, "a") as split_clusters_file,
```

Comment:
I think this should be "w" mode - when `cluster_and_write_files` is called, we expect new files to be created.